doc/source/user_guide/sparse.rst

```diff
@@ -71,130 +132,146 @@ attributes and methods that are specific to sparse data.
 This accessor is available only on data with ``SparseDtype``, and on the :class:`Series`
 class itself for creating a Series with sparse data from a scipy COO matrix with.
 
-.. _sparse.array:
 
-SparseArray
------------
+.. versionadded:: 0.25.0
+
+A ``.sparse`` accessor has been added for :class:`DataFrame` as well.
+See :ref:`api.frame.sparse` for more.
 
-``SparseArray`` is the base layer for all of the sparse indexed data
-structures. It is a 1-dimensional ndarray-like object storing only values
-distinct from the ``fill_value``:
+.. _sparse.calculation:
+
+Sparse Calculation
+------------------
+
+You can apply NumPy `ufuncs <https://docs.scipy.org/doc/numpy/reference/ufuncs.html>`_
```

Comment:
is there a reason we are recommending people work *directly* with SparseArray? the unit of computation is generally the Series, no?

Reply:
This was here before, just moved.